src/simulator/AccelerometerModule.tsx

```diff
@@ -104,6 +104,16 @@ const Gesture = ({ icon, state, enabled, onValueChange }: GestureProps) => {
     }, 500);
   }, [setActive, onValueChange, choice]);
 
+  const renderOptionText = (gesture: string): string => {
+    const translation = intl.formatMessage({
+      id: `simulator-gesture-${gesture.split(" ").join("-")}`,
+    });
+    if (translation !== gesture) {
+      gesture += ` (${translation})`;
```

Comment:
Agreed this morning that we'd put the translation first.

Reply:
[`f6037ec`](https://github.com/microbit-foundation/python-editor-v3/pull/993/commits/f6037ecd0b16471f40d369e3c9e515d5065b4660)